README.md

```diff
@@ -17,12 +17,13 @@ This analysis repository presumes that the following python packages are availab
  * dask
  * bottleneck
  * basemap
+ * pyproj
  * nco with `conda install -c conda-forge nco`
 
 You can easily install them via the conda command:
 
 ```
-conda install -c conda-forge numpy scipy matplotlib netCDF4 xarray dask bottleneck basemap nco
+conda install -c conda-forge numpy scipy matplotlib netCDF4 xarray dask bottleneck basemap nco pyproj
```

Comment:
pyproj is used to map between various projections, notably the stereographic grids used for a lot of Antarctic and Greenland data sets.